### DISCUSSION TITLE
thanks for doing this!

### DISCUSSION THREAD

**Discussion by adamfoneil** [sajalverma17/GitBranchDiffer#15] [General]
Just want to say thanks for doing this, and I look forward to giving this a try soon.
This functionality is sorely missing in Visual Studio today!

**Comment by sajalverma17** [comment]
I really appreciate that @adamfoneil ! I tried to fill that gap in Visual Studio's Git experience with this extension (and it's bugs).
Feel free to write back if you have any thoughts or suggestions😊
